cheat_sheets/command-line.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: cheat_sheet
+topic: Command line
+---
+
+### Navigating the command line
+
+| Keyboard Shortcut | Meaning                                                              |
+| ----------------- | -------------------------------------------------------------------- |
+| Arrow up/down     | Go backwards/forwards in the history of previously executed commands |
+| Tab key           | Autocomplete directory or file names                                 |
+| Ctrl + C          | Cancel whatever program is currently running in the command line     |
+| Ctrl + D          | Exit IRB or command line                                             |
+
+
+### General commands
+
+These commands are useful, no matter whether you work with Ruby or not.
+
+These are of the most important commands on the command line. You have seen all of these in the slides already, but here's a nice overview:
+
+| Command | Argument(s)                 | Meaning                   | Description                                                |
+| ------- | --------------------------- | ------------------------- | ---------------------------------------------------------- |
+| pwd     | *None*                      | "print working directory" | Shows the current working directory of your command line   |
+| ls      | *Directory name (optional)* | "list directory contents" | Lists all files and directories in a directory             |
+| cd      | *Directory name (optional)* | "change directory"        | Changes the current working directory of your command line |
+
+Here are some example usages with a description what they mean:
+
+| Command           | Meaning                                                                                              |
+| ----------------- | ---------------------------------------------------------------------------------------------------- |
+| `ls`              | List what is inside the current working directory (files and directories).                           |
+| `ls my-directory` | List what is inside my-directory, which is a directory in the current working directory.             |
+| `ls ~`            | List what is inside the current user's home directory.                                               |
+| `cd my-directory` | Change the working directory to my-directory, which is a directory in the current working directory. |
```

Comment:
Perhaps highlight _my-directory_ in the "Meaning" column?

Reply:
Good idea. I've made __my-directory__ bold, along with __home directory__ and __parent directory__.